packages/ng/root/src/style/theme/_tooltip.theme.scss

```diff
@@ -1,6 +1,6 @@
 $tooltip: (
-	background: _theme("commons.overlays.background", true),
-	color: _theme("commons.overlays.text", true),
+	background-color: _color("grey", 900, true),
```

Comment:
The key becomes "background-color", but I don't see any changes on how it used elsewhere. Is it normal ?

Reply:
yeah, if its all the same to you i'd rather keep the same theme key so this PR isnt braking changes